.gitignore

```diff
@@ -2,6 +2,9 @@
 .tags*
 /.idea/
 /browser/resources/brave_extension/
+/browser/resources/brave_sync_extension/
```

Comment:
why are we adding these to git ignore?

Reply:
because actual place for extension stuff is `src/brave/components/brave_sync/extension/` and files are copied into /browser/resources/brave_sync_extension/ while building